models/release.go

```diff
@@ -127,7 +127,9 @@ func GetRelease(repoID int64, tagName string) (*Release, error) {
 // GetReleaseByID returns release with given ID.
 func GetReleaseByID(id int64) (*Release, error) {
 	rel := new(Release)
-	has, err := x.Id(id).Get(rel)
+	has, err := x.
```

Comment:
I think this could have be left on line, it's not that long. But it's nitpicking.